<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ModelDialog</class>
 <widget class="QDialog" name="ModelDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>390</width>
    <height>446</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>5</y>
     <width>361</width>
     <height>401</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="LAYOUT_MODEL"/>
  </widget>
  <widget class="QCheckBox" name="CB_USE_ENCODER">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Encoder</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QCheckBox" name="CB_USE_REFERENCE">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>410</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Reference</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
